web/src/App.jsx

```diff
@@ -0,0 +1,37 @@
+import './App.css'
+import AboutUs from '@components/AboutUs'
+import CarouselImages from '@components/CarouselImages'
+import Faq from '@components/Faq'
+import FormLab from '@components/FormLab'
+import NavBar from '@components/NavBar'
+import News from '@components/News'
+import MapGoogle from '@components/MapGoogle'
+import Contact from '@components/Contact'
+import { Element } from 'react-scroll'
+
+function App() {
+
+  return (
+      <div className="container">
+        <NavBar></NavBar>
+        <CarouselImages></CarouselImages>
+        <Element name="aboutUs">
```

Comment:
Maintain the order of the menu with the presentation order.
1- aboutus
2- novedades
3- faq
4-contacto